src/components/Navigation/Navigation.stories.js

```diff
@@ -0,0 +1,22 @@
+import Navigation from './Navigation'
+import { BrowserRouter } from 'react-router-dom'
+
+export default {
+  title: 'Components/Navigation',
+  component: Navigation,
+}
+
+const Template = args => (
+  <BrowserRouter>
+    <Navigation {...args} />
+  </BrowserRouter>
```

Comment:
Usually for storybook and testing we use `MemoryRouter` instead of `BrowserRouter`. You can read about it here: https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/guides/testing.md